src/main/java/qna/domain/BaseEntity.java

```diff
@@ -0,0 +1,29 @@
+package qna.domain;
+
+import com.sun.istack.NotNull;
+
+import javax.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+public abstract class BaseEntity {
+    @NotNull
```

Comment:
@NotNull 어노테이션이 데이터베이스 테이블 컬럼의 요구사항을 만족하고 있을까요?
컬럼에 대한 제약조건을 걸때 @Column 을 사용도하도록 수정이 필요해보여요 🤔 
```suggestion
    @Column(nullable = false, updatable = false)
```

Reply:
@Column(nullable = false, updatable = false) 어노테이션이 어플리케이션에서도 null validation을 하나요?? 사실  @Column(nullable = false, updatable = false)과 고민하다가 어플리케이션 내에서 null validation을 하는 NotNull을 사용하면 좋을 것같아서 사용했습니다. 아니면 둘다 넣어도 괜찮을까요???

Reply:
사용해주신 NotNull 어노테이션은 어플리케이션에 사용하는 것이 아닌 문서 도구로 작성되어야 하는 어노테이션 입니다!
아마 롬복의 NotNull이나 다른 어노테이션과 헷갈리신 것 같아요!
저는 개인적으로 엔티티에서는 여러 의존성을 두는 것을 선호하지는 않는 편이여서 따로 NotNull 어노테이션을 사용하지는 않습니다 😄 
팀의 컨벤션에 따라 다를 수 있을 것 같아요 🤔 

http://daplus.net/java-%EB%A9%94%EC%86%8C%EB%93%9C-%EC%9D%B8%EC%88%98%EC%97%90%EC%84%9C-notnull-%EC%96%B4%EB%85%B8%ED%85%8C%EC%9D%B4%EC%85%98-%EC%82%AC%EC%9A%A9/